### PR TITLE
[kube-prometheus-stack] Updating value name for nodeExporter MacOS dashboard

### PR DESCRIPTION
**Pull request by nicon89**
#### What this PR does / why we need it
This PR adds new value - .Values.nodeExporterMacOS.enabled.
It's used for controlling creation of Node Exporter Mac OS Grafana dashboard.

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
n/a

#### Checklist

- [ X ] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ X ] Chart Version bumped
- [ X ] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by QuentinBisson** [comment]
This looks more like a breaking change than a patch. What value does it bring exactly?

**Comment by nicon89** [comment]
It allows one to have just a dashboard for node-exporter, if one doesn't use MacOS node-exporters, so generally for as clean installation as possible. I basically wanted to disable dashboards that I will not use - this is one of them.

File: charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def new_representer(dumper, data):
     'node-rsrc-use': ' .Values.nodeExporter.enabled',
     'node-cluster-rsrc-use': ' .Values.nodeExporter.enabled',
     'nodes': ' .Values.nodeExporter.enabled',
-    'nodes-darwin': ' .Values.nodeExporter.enabled',
+    'nodes-darwin': ' .Values.nodeExporterMacOS.enabled',
```
Comment (QuentinBisson) [inline charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py:58]: Give me some time to think, but I would rather we have some new property under nodeExporter than nodeExporterMacOS

Comment (nicon89) [reply charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py:58]: How about nodeExporterDarwin?

Comment (QuentinBisson) [reply charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py:58]: I think I would prefer to have .Values.nodeExporter.enabled and something like .Values.prometheus-node-exporter.operatingSystemFlavors (with a list containing both linux and darwin for now). That way we could eventually add windows also.


@monotek or @GMartinez-Sisti could I have your opinion?

Comment (nicon89) [reply charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py:58]: Do you want me to do it as suggested by @GMartinez-Sisti?

Comment (GMartinez-Sisti) [reply charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py:58]: Let's do it 😄

**Comment by nicon89** [comment]
Opened new pull request from new branch https://github.com/prometheus-community/helm-charts/pull/3778

File: charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py (hunk 1 above)
Comment (nicon89) [reply charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py:58]: https://github.com/prometheus-community/helm-charts/pull/3778

**Pull request closed without merging**